XMonad/Layout/NoBorders.hs

```diff
@@ -326,6 +329,9 @@ data Ambiguity
         -- ^ Focus in an empty screen does not count as ambiguous.
     | OtherIndicated
         -- ^ No borders on full when all other screens have borders.
+    | OnlyFloat
```

Comment:
I'm not sure how far down this should be in terms of ambiguity, but it feels rather ambiguous and so it was placed here.  Other opinions welcome though